website/description_issues.qmd

```diff
@@ -43,3 +43,27 @@ Document all non-obvious acronyms in the [cran-comments.md](https://usethis.r-li
 ### Details
 
 Most acronyms that are not widely known should be explained. For example, you don’t have to explain *OLS*, *SEO*, or *DNA* but explanations should be provided for: *MEFM* or *OCCDS*. If you are unsure of the acronyms common knowledge, please document all acronyms in the package.
+
+------------------------------------------------------------------------
+
+# Title Case
+
+## Problem
+
+The package Title field in the DESCRIPTION file is not in Title Case and the first letter of some or all of the words in the title are lowercase.
+
+## Solution
+
+The package Title field in the DESCRIPTION file must be in Title Case, except words like "a", "the" or "of".
+
+### Details
+
+Determining Title Case automatically can be challenging due to word significance being only known by the package author. The function `toTitleCase()` can assist with transforming sentences into Title Case from the 'tools' package.
+
+```{r}
+tools::toTitleCase("web application framework for R")
+```
+
+::: callout-note
+There are some exceptions to using Title Case for quoting longer works like book titles and shorter works like research papers - [Learn more](https://www.scribbr.com/academic-writing/title-case/#:~:text=In%20title%20case%2C%20the%20first,title%2C%20subtitle%2C%20or%20heading.).
```

Comment:
One important thing I would add as exeception: package and function names. Since R is case sensitive package names should be spelled with the correct case and in single quotes in the title. 
e.g.: One should not write 'Ggplot2' in the title even if it is wrapped in single quotes. 
However, I don't know if tool::toTitleCase() ignores words in single quotes thus solving this problem on the fly.

Reply:
I think `tools::toTitleCase` doesn't treat a package name specially

In addition, according to [writing R extensions](https://cran.r-project.org/doc/manuals/r-devel/R-exts.html#The-DESCRIPTION-file) (WRE) one must not name the package on the title (and it would be weird to name other packages in the title of your package): 
>  Do not repeat the package name: it is often used prefixed by the name. Refer to other packages and external software in single quotes, and to book titles (and similar) in double quotes. 

Maybe it is worth adding a link to that section of WRE.